server/handlers/user_handler.go

```diff
@@ -80,6 +80,42 @@ func (h *Handler) GetUsers(w http.ResponseWriter, req *http.Request, _ *models.P
 	fmt.Fprint(w, string(resp))
 }
 
+// swagger:route GET /api/identity/users/keys UserKeysAPI idGetAllUsersKeysHandler
+// Handles GET for all Keys
+//
+// # Keys can be further filtered through query parameters
+//
+// ```?order={field}``` orders on the passed field
+//
+// ```?page={page-number}``` Default page number is 0
+//
+// ```?pagesize={pagesize}``` Default pagesize is 20
+// 
+// ```?search={id|function|owner|category}``` If search is non empty then a greedy search is performed
+//
+// ```?filter={condition}```
+// responses:
+// 	200: keys
```

Comment:
This response needs to defined under `server/handlers/doc.go`.

Reply:
@leecalcote @Freedisch You are unable to run `make swagger-docs-build` as keys isn't defined under `server/handlers/doc.go`